actions/ostree_deploy_action.go

```diff
@@ -1,3 +1,46 @@
+/*
+OstreeDeploy Action
+
+Put the selected OSTree branch to target target image.
+If any preparation has been done for rootfs, it can be overwritten
+during this step.
+
+Action 'image-partition' must be called prior to OSTree deploy.
+
+Yaml syntax:
+ - action: ostree-deploy
+   repository: repository name
+   remote_repository: URL
+   branch: branch name
+   os: stateroot
+   setup-fstab: bool
+   setup-kernel-cmdline: bool
+   appendkernelcmdline: arguments
+
+Mandatory properties:
+
+- repository -- path to repository with OSTree structure; the same path is
+used by 'ostree' tool with '--repo' argument.
+This path is relative to 'artifact' directory.
+
+- os -- stateroot (aka 'osname') deployment name, as explained in:
+https://ostree.readthedocs.io/en/latest/manual/deployment/
+
+- branch -- branch name of stateroot to populate to the target image.
+
+Optional properties:
+
+- remote_repository -- URL to remote OSTree repository for pulling stateroot branch.
+Currently not implemented, please prepare local repository instead.
+
+- setup-fstab -- create fstab file for image
+
+- setup-kernel-cmdline -- if set to 'true' then add parameters listed in
+'appendkernelcmdline' property.
```

Comment:
This sentences make no sense :). setup-kernel-cmdline means adding the information from the image-partition action to the configured commandline

Reply:
Sounds better, yeah.